'afr_zerofill' does not match original declaration [-Wlto-type-mismatch]

```
afr-inode-write.h:60:1: warning: type of 'afr_zerofill' does not match original declaration [-Wlto-type-mismatch]
   60 | afr_zerofill(call_frame_t *frame, xlator_t *this, fd_t *fd, off_t offset,
      | ^
afr-inode-write.c:2205:1: note: type mismatch in parameter 5
 2205 | afr_zerofill(call_frame_t *frame, xlator_t *this, fd_t *fd, off_t offset,
      | ^
afr-inode-write.c:2205:1: note: type 'size_t' should match type 'off_t'
afr-inode-write.c:2205:1: note: 'afr_zerofill' was previously declared here
```

--- a/xlators/cluster/afr/src/afr-inode-write.c
+++ b/xlators/cluster/afr/src/afr-inode-write.c
@@ -2198,7 +2198,7 @@ afr_zerofill_wind(call_frame_t *frame, xlator_t *this, int subvol)
 
 int
 afr_zerofill(call_frame_t *frame, xlator_t *this, fd_t *fd, off_t offset,
-             size_t len, dict_t *xdata)
+             off_t len, dict_t *xdata)
 {
     afr_local_t *local = NULL;
     call_frame_t *transaction_frame = NULL;

